README.md

```diff
@@ -62,9 +70,56 @@ Outputs:
 
 * association summary statistics
 
-### To run
-
-Instructions to run each component of the pipeline using analysis runner are provided at the top of each script
+### SAIGE-QTL parameters explanation
+
+Clarifying the reasoning behind the parameters / flags used to run SAIGE-QTL.
+Most of these are (or will be) included in the official [documentation](https://weizhou0.github.io/SAIGE-QTL-doc/).
+
+Fit null model ([step 1](https://weizhou0.github.io/SAIGE-QTL-doc/docs/step1.html)):
+
+* ```pheno_file```: path specifying the location of the phenotype covariate file described above (build during part 2 of the pipeline)
+* ```cov_col_list```: string specifying the columns of the pheno_file that should be used as covariates (separated by a comma, no spaces)
+* ```sample_cov_col_list```: same as above, but specifying only, out of the columns above, the ones that are well defined at individual level (e.g., sex, age, ancestry PCs)
```

Comment:
I don't quite get why this distinction is here: when would we use cov_col_list vs sample_cov_col_list?

Reply:
sample covs list is always a subset of covs list, just specifying which are well specified at individual level, which allows to treat them more cheaply as the unique values are fewer!